### ISSUE TITLE
AddRange not firing for all Elements added

### ISSUE THREAD

**Issue by johnxy84** [jamesmontemagno/mvvm-helpers#38]
Whenever I call AddRange and pass an IEnumerable, my bound Listview only gets updated with the first element in the IEnumerable Added. If i replace NotificationMode to NotificationMode.Reset, it updates but the index of the Listview jumps to the Last item. I wonder if I'm doing something differently or wrong

**Comment by jamesmontemagno** [comment]
Do you have a code sample of how you are calling it?

**Comment by johnxy84** [comment]
Here's the Property Intialization
```
 public ObservableRangeCollection<Int> QuoteIds
        {
            get => quoteIds;
            set
            {
                SetProperty(ref quoteIds, value);
            }
        }
```
And Here's Where I add it.
`                        QuoteIds.AddRange(newquoteIds, System.Collections.Specialized.NotifyCollectionChangedAction.Reset);
`

**Comment by balivo** [comment]
@johnxy84 u don't need the SetProperty in set of QuoteIds... 

You can initialize your property using this:

`public ObservableRangeCollection<Int> QuoteIds { get; private set; } = new ObservableRangeCollection<Int>();`

**Comment by jamesmontemagno** [comment]
I would need to see a full sample project to get a better understanding of what you are seeing.

**Comment by jamesmontemagno** [comment]
I think this is fixed now in 1.6.1
